### PR TITLE
GSYE-340: Fit to limits box should be checked by default

### PR DESCRIPTION
**Pull request by ERafaelMartinez**
## Reason for the proposed changes

Ad integration test

## Proposed changes

-

**INTEGRATION_TESTS_BRANCH**=GSYE-340
**GSY_FRAMEWORK_BRANCH**=master

### CODE REVIEW

File: src/gsy_e/setup/two_sided_market/load_fitlimits_match.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                                                            hrs_of_day=range(0, 24),
+                                                            initial_buying_rate=0,
+                                                            fit_to_limit=True,
+                                                            use_market_maker_rate=True,
+                                                            )
+                         ),
+                ],
+                grid_fee_percentage=40
+            ),
+            Area("", strategy=InfiniteBusStrategy(energy_sell_rate=30, energy_buy_rate=0)
```
Comment (hannesdiedrich) [inline src/gsy_e/setup/two_sided_market/load_fitlimits_match.py:45]: Minor, but it looks better, if the area has a name IMO.

File: src/gsy_e/setup/two_sided_market/load_fitlimits_match.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from gsy_framework.constants_limits import ConstSettings
+from gsy_e.models.area import Area
+from gsy_e.models.strategy.infinite_bus import InfiniteBusStrategy
+from gsy_e.models.strategy.load_hours import LoadHoursStrategy
+
```
Comment (hannesdiedrich) [inline src/gsy_e/setup/two_sided_market/load_fitlimits_match.py:22]: Can you please add a comment here for what this setup file is used? Ideally it contains the description of the bug. So we know that this is used in the integration tests and do not delete it. Thanks!

**Review by hannesdiedrich — COMMENTED** [review]
Two comments, will approve after addressed. Thanks a lot!

**Review by mrsaemir — APPROVED** [review]

**Review by spyrostz — APPROVED** [review]
LGTM

**Pull request merged**